roles/gpu_operator_wait_deployment/tasks/main.yml

```diff
@@ -122,3 +124,79 @@
 
   - name: The GPU Operator does not correctly expose its DCGM metrics
     fail: msg="The GPU Opertor does not correctly expose the its DCGM metrics to Prometheus"
+
+- name: Check if the GPU-Operator operator metrics are exposed
+  command: oc get ServiceMonitor/gpu-operator -n openshift-operators -oname
```

Comment:
Maybe it'll be better to check if the deployed version is newer than <some version> rather than check for the existance of the ServiceMonitor. Because if someday something happens to the ServerMonitor (name change, namespace change, gets deleted because of a bundling bug or whatever), we'd silently ignore its illegitimate absence and this test won't run.

Reply:
indeed yes, good remark, I'll take that into account where finalizing this commit

more generally, my design idea would be to use `ignore_failure` to mark some tests that we know may fail (instead of silently ignoring as I did here),  and in `ci-dashboard`, show the total number of ignored tasks.
I also want to specify for a given test, how many toolbox steps are expected to fail (eg, test entitlement, test nfd, has gpu nodes --> 3) + something similar to `ignored_failures`, so that we can better visualize warnings in the dashboard. Future work, if time allows it ...